### ISSUE TITLE
from compas.utilities import gif_from_images

### ISSUE THREAD

**Issue by petrasvestartas** [compas-dev/compas_view2#206]
**Describe the bug**

compas_view2 has some issues with gif import, I think it is gone from compas2:

```bash
  File "C:\brg\2_code\compas_view2\src\compas_view2\app\app.py", line 26, in <module>
    from compas.utilities import gif_from_images
ImportError: cannot import name 'gif_from_images' from 'compas.utilities' (C:\brg\2_code\compas\src\compas\utilities\__init__.py)
```

**To Reproduce**

run this example:
https://compas.dev/compas_view2/0.10.1/examples/object/example_mesh_display.html

**Expected behavior**

Should be working.

**Desktop (please complete the following information):**

Windows VSCOde

**Comment by tomvanmele** [comment]
yes, @Licini , i had the feeling this was used somewhere in the viewer. i have a copy of it in case we want to reintegrate it directly here...

**Comment by Licini** [comment]
@tomvanmele  view2 is only meant to be used with COMPAS 1 tho. We can re-implement this separately in the new `comaps_viewer`

**Comment by tomvanmele** [comment]
only if we get the new viewer done reasonably soon :)
